spec/audited_spec_helpers.rb

```diff
@@ -17,4 +17,12 @@ def create_versions(n = 2)
     end
   end
 
+  def run_migrations(direction, migrations_paths, target_version = nil)
+    if Rails.version >= '5.2.0.0'
```

Comment:
This will fail once 5.2.0 is out, since `'5.2.0' >= '5.2.0.0' => false`.

Reply:
ouch, extra zero.
Such `Rails.version >= '5.2.0'` will not work for `.beta2`, after which change was introduced. Should I bother of exactly greater than `.beta2` or is `Rails.version >= '5.2.0'` enough?

Reply:
perhaps `Rails.version.start_with? '5.2'` for now, and change it later if needed for 6?